project_4.html

```diff
@@ -0,0 +1,524 @@
+        crossorigin="anonymous"></script>
+    <!-- Bootstrap JS v5.3.0 CDN -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.min.js"
+        integrity="sha384-fbbOQedDUMZZ5KreZpsbe1LCZPVmfTnH7ois6mU1QK+m14rQ1l2bGBq41eYeM/fS"
+        crossorigin="anonymous"></script>
+    <!-- Bootstrap Icons CSS v1.10.0 CDN-->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.0/font/bootstrap-icons.css">
+    <!-- Highlight JS v11.7.0-->
+    <link rel="stylesheet" href="./css/highlight.js/default.min.css" type="text/css">
+    <link rel="stylesheet" href="./css/highlight.js/arduino-light.min.css" type="text/css">
+    <script src="./js/highlight.min.js"></script>
+    <script>hljs.highlightAll();</script>
+</head>
+
+<body style="">
+	<!-- ======= Header ======= -->
+	<script src="./js/ops_header.js"></script>
+	<!-- End Header -->
+
+	<div class="container-xl pj-layout my-5">
+		<!-- ======= Sidebar ======= -->
+		<aside class="pj-sidebar ">
+			<nav id="project-toc" class="pj-toc d-flex flex-column px-2 pt-2 pb-4">
+				<ul class="nav flex-column">
+					<h5 class="mt-2 mb-2 mx-3">Contents</h5>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-overview">Overview</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-concepts">Concepts</a>
+						<ul class="nav">
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-arduino-nano">Arduino Nano</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-pinout">Pinout</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-potentiometer">Potentiometer</a>
+								<ul class="nav">
+									<li class="nav-item">
+										<a class="nav-link" href="#pj-heading-how-it-works-potentiometer">How it Works</a>
+									</li>
+									<li class="nav-item">
+										<a class="nav-link" href="#pj-heading-using-it-with-arduino-potentiometer">Using it with Arduino</a>
+									</li>
+								</ul>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-rgb-led">RGB LED</a>
+								<ul class="nav">
+									<li>
+										<a class="nav-link" href="#pj-heading-using-it-with-arduino-rgb">Using it with Arduino</a>
+									</li>
+								</ul>
+							</li>
+						</ul>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-software-installations">Software Installations</a>
+						<ul class="nav">
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-arduino-ide">Arduino IDE</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-ch340-driver">CH340 Driver</a>
+							</li>
+						</ul>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-requirements">Requirements</a>
```

Comment:
Please add the three subheadings for "Requirements" to the sidebar to stay consistent with the other projects pages.